### PR TITLE
feat: finminer

### PR DESCRIPTION
**Pull request by avsigaev**

### CODE REVIEW

File: finminer/FinMiner-linux-2.4.6/run.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+if [[ -d "/usr/local/nvidia/" ]]; then
+    # TODO: remove after debug
+    echo "NVIDIA libs is mounted via SONM GPU plugin, applying patch..."
+    mkdir -p /lib64/nvidia/
+    ln -s /usr/local/nvidia/lib64/libnvidia-ml.so.1 /lib64/nvidia/libnvidia-ml.so
+    export LD_LIBRARY_PATH=/usr/local/nvidia/lib64/:/lib64/nvidia/
+fi
+
+# setting finminer config with envs
+sed -i "s/wallet=WALLET/wallet=$WALLET/" config.ini
```
Comment (nikonov1101) [inline finminer/FinMiner-linux-2.4.6/run.sh:21]: Marked as required in the `task.yaml` example but not really required here. Consider empty variable checking ([example](https://github.com/sonm-io/allowed-list/blob/master/monero-amd/run.sh#L15)).

File: tasks/task_finminer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+container:
+  commit_on_stop: false
+  image: sonm/finminer:latest
+  env:
+    # Required    
+    WALLET: ""
+    # (Optional) Coin to mine. Supported values: ETH (by default), ETC, XMR, ETN, PASC. These have higher priority than the "algorithm" parameter. An arbitrary coin can also be specified.
+    COIN: ""
```
Comment (nikonov1101) [inline tasks/task_finminer.yaml:8]: Comment all non-required parameters

**Review by nikonov1101 — CHANGES_REQUESTED** [review]
BTW: do we really forced to take the `README-en.html` with us?

File: finminer/FinMiner-linux-2.4.6/run.sh (hunk 1 above)
Comment (avsigaev) [reply finminer/FinMiner-linux-2.4.6/run.sh:21]: miner fails to start and throw error in logs if wallet is empty, check in run.sh is redundant

File: tasks/task_finminer.yaml (hunk 2 above)
Comment (avsigaev) [reply tasks/task_finminer.yaml:8]: run.sh include checks for empty value for optional parameters

Comment (nikonov1101) [reply tasks/task_finminer.yaml:8]: Sure, but in other examples, we're keeping optional parameters as comments. Just for consistency. :)

**Pull request closed without merging**